man/man4/zfs.4

```diff
@@ -66,6 +66,12 @@ Set the size of the dbuf metadata cache
 .Pq Sy dbuf_metadata_cache_max_bytes
 to a log2 fraction of the target ARC size.
 .
+.It Sy dbuf_mutex_cache_shift Ns = Ns Sy 0 Pq uint
```

Comment:
The name here does not match the name of the module parameter.

Reply:
Thanks.  I've updated the code to use `dbuf_mutex_cache_shift` since I felt it was more consistent with the other module parameters.